fortress/spawn_urdf.md

```diff
@@ -8,7 +8,7 @@ More information about URDF can be found at [https://wiki.ros.org/urdf](https://
 ## Obtaining a URDF file
 
 This tutorial assumes that the reader already has a URDF file that they'd like to load into Gazebo Sim.
-If you are looking for a URDF file to use, you can use the [`rrbot.urdf`](tutorials/spawn_urdf/rrbot.urdf) that was preprocessed for you. The `rrbot` robot model is from the [`gazebo_ros_demos`](https://github.com/ros-simulation/gazebo_ros_demos) package. You could also use one of the examples listed at [https://wiki.ros.org/urdf/Examples](https://wiki.ros.org/urdf/Examples).
+If you are looking for a URDF file to use, you can use the [`rrbot.urdf`](https://github.com/khaledgabr77/docs/blob/96bd6602bb58c9a815519580a713b050634e6e60/fortress/tutorials/spawn_urdf/rrbot.urdf) that was preprocessed for you. The `rrbot` robot model is from the [`gazebo_ros_demos`](https://github.com/ros-simulation/gazebo_ros_demos) package. You could also use one of the examples listed at [https://wiki.ros.org/urdf/Examples](https://wiki.ros.org/urdf/Examples).
```

Comment:
To fix the path on Fortress, I think we need to copy `rrbot.urdf` to the `fortress` folder instead of this change here.

Reply:
Ah, got it! Moving `rrbot.urdf` to the fortress folder does make more sense. I'll go ahead and do that instead. Thanks for pointing it out!

Reply:
Do you need to `git add` the file? I don't see it in the PR.

Reply:
You can navigate below the comments to locate the commit titled [update rrbot.urdf path](https://github.com/gazebosim/docs/pull/423/commits/b8721a6563f7a6b80ff461c47693961182fad5f6).